ci/jenkins/mellanox/scripts/start_ci.sh

```diff
@@ -91,6 +95,8 @@ EOF
         return $status
     fi
 
+    docker tag antrea/antrea-ubuntu "$ANTREA_CNI_HARBOR_IMAGE"
```

Comment:
this doesn't need to run as root?

Reply:
thank you for the review, you are correct, it looks like there was a change in the machine (left over from some testing i did previously), fixed and fixed the code. Thanks.